src/redux/store.js

```diff
@@ -0,0 +1,9 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+  },
+});
+export default store;
```

Comment:
- [ ] Please you didn't add the categories reducer to the redux store,you should add so you can use it in this store, it should look something like this const store = configureStore({
  reducer: { books: booksReducer, categories: categoriesReducer },
})

![image](https://github.com/Rimuru-Tempest-26/Bookstore/assets/74744787/70c7ea35-5a92-44c7-923c-184a7d8c5088)

Reply:
Fixed.